src/Tasks/Microsoft.NET.Build.Tasks/ProcessFrameworkReferences.cs

```diff
@@ -412,19 +391,6 @@ public KnownFrameworkReference(ITaskItem item)
 
             public string Profile => _item.GetMetadata("Profile");
 
-            public string LegacyFrameworkPackages
-            {
-                get
-                {
-                    var packages = _item.GetMetadata("LegacyFrameworkPackages");
```

Comment:
Do you have a change queued to remove this metadata from bundled versions?

Reply:
I now have this: https://github.com/dotnet/core-sdk/pull/2970